src/turnrest.c

```diff
@@ -94,6 +98,12 @@ void janus_turnrest_set_backend(const char *server, const char *key, const char
 				api_http_get = FALSE;
 			}
 		}
+		if(unixpath != NULL) {
+			if(access(unixpath, R_OK | W_OK) != 0) {
+				JANUS_LOG(LOG_WARN, "TURN REST API unix socket '%s' was not accessible by the service during startup\n", unixpath);
```

Comment:
I account for the situation that the socket is not yet ready at janus startup, eg. because related service is still initializing. The socket must be accessible only at the time of the request, and then, appropriate error will be generated by curl.

Reply:
While getting acquainted with janues in more detail, I came across a comment on `ignore_unreachable_ice_server` configuration variable.

> In case you want to allow Janus to start even if the configured STUN or TURN
> server is unreachable, you can set 'ignore_unreachable_ice_server' to true.
> WARNING: We do not recommend to ignore reachability problems, particularly
> if you run Janus in the cloud. Before enabling this flag, make sure your
> system is correctly configured and Janus starts after the network layer of
> your machine is ready. Note that Linux distributions offer such directives.
> You could use the following directive in systemd: 'After=network-online.target'

This gives me insight, that maybe one of the janus project assumption is that, by default, the process will fail if it recognizes that its runtime environment is not yet ready or malfunctioning. If it is so, I will rewrite this aspect.

Similarly to cited above note on ignore_unreachable_ice_server, adding a note on 
`Requires`/`Wants`/`After` systemd unit option may be considered.